### PR TITLE
Changed SDK initialization

### PR DESCRIPTION
**Pull request by vmanilo**
@drewlr please have a look

after user do `import remo`
next step could be `remo.init()` or `remo.init_sdk(SDK(URL, user_email, user_password))`
after that all everything works the same

### CODE REVIEW

File: remo/__init__.py [hunk 1]
```diff
@@ -1,163 +1,41 @@
...
-        python -m remo_app init
-        
-        """.format(
-                cfg_path
-            )
-        )
-
-
-__init__()
+def connect_sdk(server: str, email: str, password: str, viewer: str = 'browser'):
```
Comment (drewlr) [inline remo/__init__.py:18]: @vmanilo do we want to use the default viewer from the config file?

Or are you thinking that if installing the sdk alone, user won't have the config file?

File: remo/config.py [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+REMO_HOME = os.getenv('REMO_HOME', str(Path.home().joinpath('.remo')))
 
 
 class Config:
-    __slots__ = ['port', 'server', 'user_name', 'user_email', 'user_password', 'conda_env', 'viewer']
+    """
+    Remo Config
+
+    This class is used to initialise various settings.
+    #TODO: add description of how those are initiliased from code vs from config file
```
Comment (drewlr) [inline remo/config.py:13]: @vmanilo not clear here how we use these values. Do we try to read from a config, and if not present, use these values here?

File: remo/__init__.py (hunk 1 above)
Comment (vmanilo) [reply remo/__init__.py:18]: @drewlr I think it's good how you did

if user don't have config file, `init()` will fail, but user can use `connect_sdk()`

File: remo/config.py (hunk 2 above)
Comment (vmanilo) [reply remo/config.py:13]: yes, we have some default values, if they missing in config

but I don't expect that user need to use this class

File: remo/__init__.py [hunk 3]
```diff
@@ -5,26 +5,22 @@
...
-    """
-    from .config import Config
-    config = Config.load()
-    connect_sdk(config.server_url(), config.user_email, config.user_password, config.viewer)
-
-
-def connect_sdk(server: str, email: str, password: str, viewer: str = 'browser'):
+def connect(server: str = '', email: str = '', password: str = '', viewer: str = 'browser'):
     """
     Connect to a remote running remo server.
```
Comment (drewlr) [inline remo/__init__.py:10]:
```suggestion
    Connect to a remo server.
```

File: remo/__init__.py [hunk 4]
```diff
@@ -5,26 +5,22 @@
...
+    By default connects to a local running remo server.
```
Comment (drewlr) [inline remo/__init__.py:11]:
```suggestion
    If no parameters are passed, it connects to a local running remo server. To connect to a remote remo, specify connection details.
```

**Pull request merged**